tests/Microsoft.Bot.Builder.Azure.Tests/AzureBlobStorageTests.cs

```diff
@@ -191,6 +191,15 @@ public async Task TestConversationStateBlobStorage_TypeNameHandlingDefault()
             }
         }
 
+        [TestMethod]
+        public async Task UpdateObjectTest_AsJObjects_TypeNameHandlingNone()
```

Comment:
nit: can we avoid using underscores in method names? (PascalCase should be enough)